module1/task2.py

```diff
@@ -0,0 +1,12 @@
+numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
+max_value = numbers[0]
+min_value = numbers[0]
+for n in numbers:
+    try:
+        temp = int(n)
```

Comment:
It's not correct for float numbers. In this case your min number is -2, not -2.5

Reply:
According to the task we have to convert the elements from the list to int - https://git.epam.com/Oleksandr_Grytsenko/python_aws_cdp/blob/master/02_min_and_max.md